devmaster/modules/devshop/devshop_projects/inc/create/step-3.inc

```diff
@@ -244,7 +244,7 @@ function devshop_project_create_step_environments_submit(&$form, &$form_state) {
   }
 
   // For all removed platforms, trigger a delete task
-  $removed_environments = array_diff_key($project->environments, $environments);
+  $removed_environments = array_diff_key($project->environments ?? [], $environments);
```

Comment:
I'm not familiar with the `??` syntax, can you point me at some documentation about it?

What version of PHP does it require?

Reply:
Sure, thanks for asking. It is available since PHP 7.0. You can find more information here: https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op